lib/vector-core/src/event/mod.rs

```diff
@@ -11,14 +11,15 @@ use vector_common::EventDataEq;
 
 use crate::ByteSizeOf;
 pub use ::value::Value;
-pub use array::{EventArray, EventContainer, LogArray, MetricArray};
+pub use array::{into_event_stream, EventArray, EventContainer, LogArray, MetricArray, TraceArray};
 pub use finalization::{
     BatchNotifier, BatchStatus, BatchStatusReceiver, EventFinalizer, EventFinalizers, EventStatus,
     Finalizable,
 };
 pub use log_event::LogEvent;
 pub use metadata::{EventMetadata, WithMetadata};
 pub use metric::{Metric, MetricKind, MetricValue, StatisticKind};
+pub use r#ref::{EventMutRef, EventRef};
```

Comment:
Wait, what syntax is this `r#ref::`?

Reply:
That is a new(ish) raw identifier literal, which was added a few versions back to allow for using identifiers that are named the same as a reserved word.

Reply:
Dang.